package.json

```diff
@@ -16,8 +16,9 @@
     "lint:js": "eslint \"**/*.js\" --ignore-pattern node_modules/",
     "lint:css": "sass-lint 'app/**/*.scss' -v -q",
     "nuke": "npm run clean && rm -rf node_modules/",
-    "start:dev": "cross-env LOGGER_LEVEL=info babel-node ./app/server",
+    "start:dev": "concurrently \"npm run start:redis\" \"cross-env LOGGER_LEVEL=info babel-node ./app/server\"",
     "start": "cross-env NODE_ENV=production npm run build && cross-env NODE_ENV=production babel-node ./app/server",
```

Comment:
do we use this command in production? if so will it require redis? and also babel-node should be removed

Reply:
I added this to the dev script to make it easier for running locally. For production the platform guys will have to make sure it's up and running.

Don't we still need babel-node? when we run locally we don't do the transpiling of the JS like we do when we run it in the docker container.

Reply:
babel-node should not be used in production - I will take a look at what we do elsewhere and bring this repo up to date at some point. Tech debt 😆

Reply:
mmm this isn't the production script. this it the dev script for running the app locally. We no longer use babel-node in our hosted environments.

Reply:
ok so it is not used in production then

Reply:
no I don't believe so. As I understand we use the dockerfile to run in production and use that script to run our app. In fact the `start` script is actually redundant. I will get rid of that.